eddy/ui/session.py

```diff
@@ -3482,7 +3488,8 @@ def closeEvent(self, closeEvent: QtGui.QCloseEvent) -> None:
             if save:
                 self.sgnSaveProject.emit()
             else:
-                self.sgnNoSaveProject.emit()
+                if notSave:
+                    self.sgnNoSaveProject.emit()
```

Comment:
Quale condizione serve a evitare il test che non mi è chiaro?

Reply:
Qui praticamente era il motivo per cui non salvava le importazioni, o meglio le rimuoveva, nel caso in cui non venissero salvate alla chiusura del progetto. Comunque ho rimosso `notSave` e spostato l'emissione del segnale nel caso in cui si sceglie esplicitamente di non salvare.

Reply:
Ok ora è chiaro, praticamente il segnale veniva emesso quando si chiudeva la sessione ed il progetto era in uno stato "pulito", senza sollevare il dialogo di conferma, in quanto `save` era `False`.